frontend/config/env.ts

```diff
@@ -31,6 +31,9 @@ export const MATOMO_URL = process.env.NEXT_PUBLIC_MATOMO_URL;
 
 export const PIWIK_ID = getPiwikIDForEnv(process.env.NEXT_PUBLIC_PIWIK_ENV);
 
+export const DISABLE_SENTRY_SAMPLING =
+   process.env.DISABLE_SENTRY_SAMPLING === 'true';
```

Comment:
Minor: I find it easier to read the positive form in all cases:

```suggestion
export const SENTRY_SAMPLING_ENABLED =
   process.env.SENTRY_SAMPLING === 'true';
```

Reply:
How about `SENTRY_SAMPLING_OVERRIDE=1.0` instead? Clear and tunable.

Reply:
What's the use case for a custom value, as opposed to "force it to be on" flag?

I really think that the production value should be hardcoded for visibility and history, so I'm worried that an env var custom value might get used in production. What do you think about only allowing the `SENTRY_SAMPLING_OVERRIDE` to be usable in preview and development environments?

Reply:
> I really think that the production value should be hardcoded for visibility and history

I think @masonmcelvain has a good point here. I've gone and looked up the error sample rate in the PHP app multiple times, and the code is where I expect to find an answer. Also, I don't really see utility in the area between "a small sample" and 100%.